webapp-django/crashstats/crashstats/utils.py

```diff
@@ -272,6 +272,7 @@ def get_recent_versions_for_product(product):
             'product': product,
             'version': major_to_versions[major_key][0],
             'is_featured': True,
+            'has_builds': False
```

Comment:
This was the other half of the issue. The  faux versions needed a `has_builds` property that topcrashers uses.